scripts/sync_makefiles.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -uo pipefail
+
+# GITHUB_TOKEN required scope: repo.repo_public
+
+git_mail="prometheus-team@googlegroups.com"
+git_user="prombot"
+
+git_user="paulfantom"
+
+branch="makefile_common"
+commit_msg="makefile: update Makefile.common with newer version"
+pr_title="Synchronize Makefile.common from prometheus/prometheus"
+pr_msg="Propagating changes from master Makefile.common located in prometheus/prometheus."
+org="prometheus"
+
+if [ -z "${GITHUB_TOKEN}" ]; then
+	echo -e "\e[31mGitHub token (GITHUB_TOKEN) not set. Terminating.\e[0m"
+	exit 1
+fi
+
+# Go to the root of the repo
+cd "$(git rev-parse --show-cdup)"
+
+source_makefile="$(pwd)/Makefile.common"
+source_checksum="$(sha256sum Makefile.common | cut -d' ' -f1)"
+
+tmp_dir=$(mktemp -d)
+trap "rm -rf ${tmp_dir}" EXIT
+
+# iterate over all repositories in ${org}
+curl --retry 5 --silent -u "${git_user}:${GITHUB_TOKEN}" https://api.github.com/users/${org}/repos 2>/dev/null | jq -r '.[] | select( .name != "prometheus" ) | .name' | sed 's/"//g' | while read -r; do
```

Comment:
I think the `jq -r` replaces the need for `sed 's/"//g'`.

Reply:
Yup, you're right :+1: